views/say.handlebars

```diff
@@ -0,0 +1,7 @@
+<h1>{{title}}</h1>
+<form method="post">
+    <input type="text" name="message" placeholder="message" maxlength="500" value="" />
+    <input type="hidden" name="password" value="{{data/password}}" />
+    <button>Send</button>
+</form>
+{{data/statusText}}
```

Comment:
We should probably move the status text to the top - given how many ENV vars we have, it is easy to not notice it at the bottom. It's not like anyone else uses the dashboard, but it is still nice to see the status easily. Mostly noting to myself - been thinking about it for a while